rpc/src/v1/types/account_info.rs

```diff
@@ -41,3 +43,19 @@ pub struct HwAccountInfo {
 	/// Device manufacturer.
 	pub manufacturer: String,
 }
+
+/// account derived from a signature
+/// as well as information that tells if it is valid for
+/// the current chain
+#[derive(Debug, Clone, Serialize, PartialEq, Eq)]
```

Comment:
Do we really need all these derives? It's hard to tell from the code in this PR if that is the case. Thanks!